test/image/mocks/8.json

```diff
@@ -1256,7 +1256,6 @@
             "size": 12,
             "color": "grey"
         },
-        "showlegend": true,
```

Comment:
First set to `true` here!

Reply:
But then set to `false` few lines below.

Reply:
I'm surprised that's even legal JSON!